untitled1/src/main/com/aca/homework/week7/car/tax/CarType.java

```diff
@@ -0,0 +1,23 @@
+package com.aca.homework.week7.car.tax;
+
+public enum CarType {
+    ELECTRIC(2015),
```

Comment:
why did you hardcode the model year? we may have an electric car produced on 2017